zwave_js/rootfs/etc/services.d/zwave_js/run

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/with-contenv bashio
+# ==============================================================================
+# Start OpenZWave service for zwave radio
+# ==============================================================================
+SERIAL_DEVICE=$(bashio::config 'device')
+
+# Emulate serial Hardware for test & development
+if bashio::config.true 'emulate_hardware'; then
+    SERIAL_DEVICE="--mock"
+fi
+
+# Send out discovery information to Home Assistant
+./discovery &
+
+# Run daemon
+exec ts-node /usr/src/zwavejs/src/bin/server.ts ${SERIAL_DEVICE} --config /etc/zwave_config.json
```

Comment:
There should be a script installed when installing the server with npm, `zwave-server`. We should be able to call that instead of using the full path.

Reply:
`ts-node zwave-server` ... ?

Reply:
Here's an example:
https://github.com/kpine/dockerfiles/blob/3fde5f9bf82fba19af21984fd0197dae1b4d348d/zwave-js-server/docker-entrypoint.sh#L18

It may be enough to just call `zwave-server` depending on if the script is available on the path after the install. Normally global npm installs should make scripts available on the path, I think.